python/bunsen/codes/__init__.py

```diff
@@ -258,10 +279,10 @@ def get_values(self, url=None, version=None):
         df = DataFrame(self._jvalue_sets.getValues(), self._spark_session)
 
         if  url is not None:
-            df = df.where(df.valueSetUri == functions.lit(url))
+            df = df.where("valueseturi = '" + url + "'")
 
         if version is not None:
-            df = df.where(df.valueSetVersion == functions.lit(url))
+            df = df.where("valuesetversion = '" + version + "'")
```

Comment:
These filtering operations weren't previously getting tested, but since adding a test that uses these arguments, I noticed that Pyspark isn't properly recognizing the column attribute syntax for any case of the field name:

When using lowercase `df.valueseturi`:
```
self = DataFrame[system: string, value: string, valueSetUri: string, valueSetVersion: string, version: string]
name = 'valueseturi'

    @since(1.3)
    def __getattr__(self, name):
        """Returns the :class:`Column` denoted by ``name``.

            >>> df.select(df.age).collect()
            [Row(age=2), Row(age=5)]
            """
        if name not in self.columns:
            raise AttributeError(
>               "'%s' object has no attribute '%s'" % (self.__class__.__name__, name))
E           AttributeError: 'DataFrame' object has no attribute 'valueseturi'
```

When using uppercase `df.valueSetUri`:
```
self = DataFrame[system: string, version: string, value: string, valueseturi: string, valuesetversion: string]
name = 'valueSetUri'

    @since(1.3)
    def __getattr__(self, name):
        """Returns the :class:`Column` denoted by ``name``.

            >>> df.select(df.age).collect()
            [Row(age=2), Row(age=5)]
            """
        if name not in self.columns:
            raise AttributeError(
>               "'%s' object has no attribute '%s'" % (self.__class__.__name__, name))
E           AttributeError: 'DataFrame' object has no attribute 'valueSetUri'
```